### PR TITLE
notify ops when job succeeds

### PR DESCRIPTION
**Pull request by camallen**
_Please request review from `@zooniverse/frontend` team. If PR is related to design, please request review from `@beckyrother` in addition._ 

Package: all/none - ops related

notify slack ops channel when this job succeeds, get some visibility for deploy success

# Review Checklist

## General

- [ ] Are the tests passing locally and on Travis?
- [ ] Is the documentation up to date?

## Components
- [ ] Has a storybook story been created or updated?
- [ ] Is the component accessible? 
  - [ ] Can it be used with a screen reader? [BBC guide to testing with VoiceOver](https://bbc.github.io/accessibility-news-and-you/accessibility-and-testing-with-voiceover-os.html)
  - [ ] Can it be used from the keyboard? [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
  - [ ] Is it passing accessibility checks in the storybook?

## Apps

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `yarn panic && yarn bootstrap` or `docker-compose up --build` and app works as expected?

## Publishing

- [ ] Is the changelog updated?
- [ ] Are the dependencies updated for apps and libraries that are using the newly published library?

## Post-merging

- [ ] Did the app deploy to https://frontend.preview.zooniverse.org/projects/:project-name/:owner or https://frontend.preview.zooniverse.org/about?
- [ ] Is the new feature working or bug now fixed?
  - [ ] Is there a Talk or blog post written to announce the new feature(s)?
- [ ] Is the design working across browsers (Firefox, Chrome, Edge, Safari) and mobile?
  - [ ] Is this approved by our designer?
- [ ] Is this ready for production deployment?

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -125,6 +125,13 @@ pipeline {
   }
 
   post {
+    success {
+      slackSend (
+        color: '#00FF00',
+        message: "SUCCESSFUL: Job '${JOB_NAME} [${BUILD_NUMBER}]' (${BUILD_URL})",
+        channel: "#frontend-rewrite"
```
Comment (camallen) [inline Jenkinsfile:132]: I think we should move these to the #ops channel

File: Jenkinsfile [hunk 2]
```diff
@@ -125,6 +125,13 @@ pipeline {
   }
 
   post {
+    success {
```
Comment (camallen) [inline Jenkinsfile:128]: I think this should only notify if this is a deploy (master / production-release job), https://github.com/zooniverse/panoptes/blob/4886b71d1857afbc1a5970d122b4a256bf550819/Jenkinsfile#L141-L151

**Review by eatyourgreens — APPROVED** [review]

**Comment by eatyourgreens** [comment]
This PR has been spamming Slack over the weekend. image

**Comment by camallen** [comment]
I've modified the code to only notify #frontend-rebuild channel on failed branch builds. Deploy branches (master / production-release) will notify to #ops channel on build / deploy success or failure.

**Pull request merged**